DEVELOPMENT.md

```diff
@@ -208,8 +208,8 @@ ko delete -f config/post-install/default-domain.yaml --ignore-not-found
 ko apply -f config/post-install/default-domain.yaml
 ```
 
-The above step is equivalent to applying the `serving.yaml` for released
-versions of Knative Serving.
+The above step is equivalent to applying the `serving-crds.yaml` and
```

Comment:
Installing `serving-crds.yaml`+`serving-core.yaml` is not equivalent to installing `serving.yaml` as it does not install the Istio controller, so the below `kubectl -n knative-serving get pods` command won't get the `networking-istio-6755db495d-wtj4d` pod.